### PR TITLE
:book: Tweak Best Practices badge description to clarify things

### PR DESCRIPTION
**Pull request by david-a-wheeler**
#### What kind of change does this PR introduce?

(Is it a bug fix, feature, docs update, something else?)

- [ ] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the new behavior (if this is a feature change)?**

- [ ] Tests for the changes have been added (for bug fixes/features)

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note

```

### CODE REVIEW

**Review by laurentsimon — APPROVED** [review]

**Comment by david-a-wheeler** [comment]
Drat, I started on a local copy of main instead of the real tip. I also didn't edit the YAML file. Give me a moment to fix that!

File: README.md [hunk 1]
```diff
@@ -433,12 +433,13 @@ These may be specified with the `--format` flag. For example, `--format=json`.
 
 The following checks are all run against the target project by default:
 
+[CII-Best-Practices](docs/checks.md#cii-best-practices)         | Has the project earned an [OpenSSF (formerly CII) Best Practices Badge](https://bestpractices.coreinfrastructure.org) at the passing, silver, or gold level?                                                                                                                                                                                                                         | Low  | PAT, GITHUB_TOKEN   |
```
Comment (spencerschrock) [inline README.md:436]: This looks like a copy paste error?

**Review by spencerschrock — COMMENTED** [review]
Left 1 comment, looks good otherwise.

File: README.md (hunk 1 above)
Comment (david-a-wheeler) [reply README.md:436]: Hmm, only 1 intended. Let me try to tweak that...!!

Comment (david-a-wheeler) [reply README.md]: I think I fixed it now, thanks!

**Pull request merged**